Library/Homebrew/.rubocop_todo.yml

```diff
@@ -116,6 +116,11 @@ Style/MultilineBlockChain:
     - 'diagnostic.rb'
     - 'test/test_patching.rb'
 
+# FIXME: temporarily added to allow CI to pass
+Style/MultilineIfModifier:
+  Exclude:
+    - 'test/test_os_mac_bottle_tag.rb'
+
```

Comment:
You can get rid of this exception and instead change
```ruby
end if OS.mac?
```
to
```ruby
end
```
This test is now skipped by default by `brew tests`, I'm guessing based on its name prefix `os_mac`.